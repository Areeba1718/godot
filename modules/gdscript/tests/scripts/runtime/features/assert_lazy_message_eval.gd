# GH-78610

func get_condition(x: bool) -> bool:
	print("get_condition(%s)" % x)
	return x

func get_message(x: bool) -> String:
	print("get_message(%s)" % x)
	return "message"

func test():
	# Condition is true, so message is not evaluated.
	assert(get_condition(true), get_message(true))
