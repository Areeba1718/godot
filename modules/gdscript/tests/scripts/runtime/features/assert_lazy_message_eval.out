GDTEST_OK
get_condition(true)
